### DISCUSSION TITLE
.opus files

### DISCUSSION THREAD

**Discussion by renatoianhez** [chidiwilliams/buzz#738] [General]
Are there plans to support .opus files generated by WhatsApp?

**Comment by raivisdejus** [comment]
https://github.com/chidiwilliams/buzz/pull/750 will add `*.opus` to the supported list of audio files.

Technically Buzz is able to process .opus files also right now, to select them select "All files" in the file chooser and you will be able to select your Opus files
![attels](https://github.com/chidiwilliams/buzz/assets/5319134/f9806bf4-a08c-4db9-af07-bbd15f27a7a8)

**Comment by renatoianhez** [comment]
Great!!!
